gpkit/constraints/tight.py

```diff
@@ -3,6 +3,11 @@
 from ..nomials import PosynomialInequality, SignomialInequality
 
 
+class TightnessError(Exception):
+    "Raised when linked variables have differing values."
+    pass
```

Comment:
Maybe! The disadvantage of that would be that catching ValueError (which we/cvxopt sometimes use to indicate e.g. solver failure) would also catch this kind of error; being able to tell the difference between those two types of error would be useful.